tasks/test-project/tasks.js

```diff
@@ -127,6 +127,21 @@ async function webTasks(outputPath, { linkWithLatestFwBuild, verbose }) {
           )
         },
       },
+      {
+        title: 'Creating MDX Storybook stories',
+        task: () => {
+          const redwoodMdxStoryContent = fs.readFileSync(
```

Comment:
Possible to move the mdx stuff into the codemods folder? Codemods in this context means "modify the source of a blank rw project in someway" - not just modifying the files that are already there.

Reply:
Understood - just asking if we could move the file to the codemods folder, it’s just weird that all the other mods are in the folder but this isn’t.